src/App.js

```diff
@@ -10,29 +10,47 @@ const useStyle = makeStyles((theme) =>
     bar: {
       padding: theme.spacing(1),
     },
+    titleDesktop: {
+      fontSize: '45px',
```

Comment:
Missed this in a previous review, you could have used `rem` instead of `px` here but not critical.

Reply:
I see.
I changed all the `fontSize` to `rem`.